src/placeos-rest-api/controllers/api_keys.cr

```diff
@@ -0,0 +1,62 @@
+require "./application"
+
+module PlaceOS::Api
+  class ApiKeys < Application
+    base "/api/engine/v2/api_keys/"
+
+    before_action :check_admin
+    before_action :body, only: [:create, :update, :update_alt]
+
+    getter current_api_key : Model::ApiKey { find_api_key }
+
+    def index
```

Comment:
We’ll need to add the model into rubber-soul as well

Reply:
should be in the next nightly

Reply:
The class needs to be included here  https://github.com/PlaceOS/rubber-soul/blob/754cb9476ab67c74468104d24de80f7a3c5b9e07/src/config.cr#L9